### PR TITLE
Completion

### PR DESCRIPTION
**Pull request by martinRenou**
completion

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

Needs a lint pass for the build to pass: https://github.com/jtpio/jupyterlite/pull/142/checks?check_run_id=2821924548

**Comment by martinRenou** [comment]
Thanks for the heads up :)

File: packages/pyolite-kernel/src/kernel.ts [hunk 1]
```diff
@@ -245,21 +237,12 @@ export class PyoliteKernel extends BaseKernel implements IKernel {
    *
    * @param msg The message to send to the worker.
    */
-  private async _sendWorkerMessage(msg: any): Promise<any> {
+  private async _sendWorkerMessage(type: string, data: any): Promise<any> {
```
Comment (jtpio) [inline packages/pyolite-kernel/src/kernel.ts:240]: Let's update the docstring above since the parameters have changed?

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**